src/dotnet/APIView/APIViewWeb/LeanModels/ReviewRevisionPageModels.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using ApiView;
+using APIView;
+using APIViewWeb.Models;
+
+namespace APIViewWeb.LeanModels
+{
+    public class ReviewContentModel
+    {
+        public ReviewListItemModel Review { get; set; }
+        public NavigationItem[] Navigation { get; set; }
+        public CodeLineModel[] codeLines { get; set; }
+        public Dictionary<string, List<ReviewRevisionListItemModel>> ReviewRevisions { get; set; }
+        public ReviewRevisionListItemModel ActiveRevision { get; set; }
```

Comment:
ActiveRevision is one of the item in ReviewRevision so it will be duplicate if this gets converted to JSON. Can we store the ID of the active revision instead?

Reply:
That should work. I think I did this for convenience though to avoid the extra look up.